libPhoneNumber/NBPhoneNumberOfflineGeocoder.m

```diff
@@ -0,0 +1,173 @@
+//
+//  NBPhoneNumberOfflineGeocoder.m
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 ohtalk.me. All rights reserved.
+//
+
+#import "NBPhoneNumberOfflineGeocoder.h"
+
+@implementation NBPhoneNumberOfflineGeocoder {
+ @private
+  NBPhoneNumberUtil *phoneUtil;
+  NBGeocoderMetadataHelper *geocoderHelper;
```

Comment:
geocoderHelper -> metadataHelper? Also see comment below about making this NSCache instead

Reply:
Yup, this has been updated as well. thanks.